### DISCUSSION TITLE
Generate Silence window does not minimize to taskbar

### DISCUSSION THREAD

**Discussion by Djard** [audacity/audacity#4745] [Far future]
I clicked on "Generate" then "Silence" and accidentally minimized the pop-up window. It did not appear in Windows' taskbar. In order to resume work, I had to press Ctrl+Shift+Esc and force Audacity to close. I am able to replicate the problem. Is this a bug, or is the issue peculiar to my system?

**Comment by LWinterberg** [comment]
For me, clicking on another program in the taskbar and then clicking on the audacity icon in the task bar brings back the window

**Reply by Djard** [reply]
Thanks. Your workaround is better than mine. Hopefully the issue will be fixed in the next release.
